### ISSUE TITLE
File insertion error. Nothing after inserting the file. 

### ISSUE THREAD

**Issue by YozhikR** [ckeditor/strapi-plugin-ckeditor#92]
Does anyone know how to solve this? 
![image](https://github.com/ckeditor/strapi-plugin-ckeditor/assets/44803662/47178a7a-e52b-4a2f-9669-cfe4d4ef7786)
![image](https://github.com/ckeditor/strapi-plugin-ckeditor/assets/44803662/d0aaf597-ebad-472c-8799-3cb29b7204bd)

**Comment by Mgsy** [comment]
Currently, CKEditor 5 supports inserting only images. I'm curious what behavior you would like to see here (inserting a link to the PDF, rendering PDF)?

**Comment by YozhikR** [comment]
Yes, as in RichText field. It's just that it supports markdown, not html.
